### DISCUSSION TITLE
MacOS setuptools issue for Briefcase 0.3.19

### DISCUSSION THREAD

**Discussion by GeoArkadeep** [beeware/briefcase#1981] [General]
On using briefcase create on an existing project, I ran into an issue where the command failed due to build/appname/macos/app/app_packages.x86_64/setuptools/_distutils/_vendor/packaging-24.0.dist-info/WHEEL being reportedly missing.
After looking through the logs, it seems setuptools 74.0.0 was being installed/used.
I managed to workaround the issue by manually listing setuptools==65.5.0 in my pyproject.toml though my app does not have this dependency directly.
Perhaps something needs to be fixed somewhere? Posting this to help out in case someone else runs into this issue.

**Comment by rmartin16** [comment]
Thank you for letting us know; we have [resolved](https://github.com/beeware/briefcase/pull/1972) this in Briefcase and the next release will include the fix. In the meantime, if you use `setuptools<73.0.0`, you can avoid the issue.
